src/headers/sec.h

```diff
@@ -170,6 +171,15 @@ int OS_DeleteSocket(keystore * keys, int sock);
  */
 int w_get_agent_net_protocol_from_keystore(keystore * keys, const char * agent_id);
 
+/**
+ * @brief Receives a keyentry structure and returns the SHA1 value of the agent's key.
+ *
+ * @param key_entry The agent's key structure
+ * @param output The variable where the hashed key will be stored
+ * @retval Returns OS_SUCCESS on success or OS_INVALID on error.
+ **/
+int w_auth_hash_key(keyentry *key_entry, os_sha1 output);
```

Comment:
**w_auth** is a prefix for methods inside **auth.c** module.
As **keys.c** doesn´t have its own prefix, the correct name of this method could be:
`w_get_key_hash()`

Reply:
You are right, thank you !